openassessment/locale/fr/LC_MESSAGES/django.po

```diff
@@ -1097,8 +1097,8 @@ msgid "Your Submission Status"
 msgstr "Statut de votre envoi"
 
 #: openassessment/templates/openassessmentblock/response/oa_response.html:142
-msgid "We could not upload files"
-msgstr ""
+msgid "We could not upload files."
+msgstr "Nous n'avons pas pu télécharger le fichiers."
```

Comment:
Nous n'avons pas pu télécharger **les** fichiers.

don't forget to recompile _openassessment/locale/fr/LC_MESSAGES/django.mo_ afterwards